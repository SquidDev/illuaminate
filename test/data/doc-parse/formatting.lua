--- This is a long doc comment to test some formatting.
--
-- This @{print|is a reference to print}, and @{read.io} is a standalone reference.
--
-- We accept #FFF and #ff00aa colour formatting codes. #999999
