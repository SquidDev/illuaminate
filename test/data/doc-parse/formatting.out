((source formatting.lua[1:1-5:62])
 (description
  "This is a long doc comment to test some formatting.\n\nThis <illuaminate:ref link='print'>is a reference to print</illuaminate:ref>, and <illuaminate:ref link='read.io'>read.io</illuaminate:ref> is a standalone reference.\n\nWe accept <illuaminate:colour colour='FFF'>\\#FFF</illuaminate:colour> and <illuaminate:colour colour='ff00aa'>\\#ff00aa</illuaminate:colour> colour formatting codes. <illuaminate:colour colour='999999'>\\#999999</illuaminate:colour>\n\n"))
